browser/admin/src/integrator/WordBook.ts

```diff
@@ -112,12 +115,12 @@ class WordBook {
 		{
 			value: 'positive',
 			heading: 'Standard dictionary',
```

Comment:
This was omitted by mistake _('Standard dictionary')

Reply:
Thanks :=)